### DISCUSSION TITLE
p200 リスト18.22の `WithArgs` メソッドの引数について

### DISCUSSION THREAD

**Discussion by halllllll** [budougumi0617/go_todo_app#76] [Q&A]
素晴らしい書籍をありがとうございます。読み進めていて気になった箇所があります。リスト18.22で`okTask`の各フィールドをこのように指定してありますが、

```store/task_test.go
okTask := &entity.Task{
    Title:    "ok task",
    Status:   "todo",
    Created:  c.Now(),
    Modified: c.Now(),
}
```

`WithArgs`メソッドでは第3,第4引数で、再度`c.Now()`を与えているのはなぜでしょうか？

https://github.com/budougumi0617/go_todo_app/blob/b0235f6c3797fe2f68bc3106d86a6fe2ae00b7b1/_chapter18/section74/store/task_test.go#L102

**Comment by budougumi0617** [comment]
- https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.11

ご指摘ありがとうございます。たしかに `okTask.Created` などを利用するほうが適切ですね…。コードと正誤表を修正しました！

**Reply by halllllll** [reply]
ご対応ありがとうございます！書籍のレベル感にまだ自分の理解が及ばないところもあり、自信がなかったので些末ではありますが質問した次第でした。とても勉強になっております。

**Reply by budougumi0617** [reply]
そういってもらえるととても嬉しいです。また何か疑問点あれば投稿していただけると幸いです！！
